lib/services/platform-service.ts

```diff
@@ -305,6 +315,10 @@ export class PlatformService implements IPlatformService {
 			}
 		}).future<void>()();
 	}
+	
+	private isPlatformInstalled(platform: string): IFuture<boolean> {
```

Comment:
It seems that platform name is not always lower-cased. Consider adding toLowerCase().